### DISCUSSION TITLE
Lack of IPv6 support

### DISCUSSION THREAD

**Discussion by huaracheguarache** [vimeo/player.js#995] [Feature requests]
Vimeo doesn't support IPv6, at least for the following domains:

- vimeo.com
- player.vimeo.com
- f.vimeocdn.com
- i.vimeocdn.com

We're in 2023 and [Google's statistic on IPv6 adoption](https://www.google.com/intl/en/ipv6/statistics.html) shows that more than 40% of users access Google's services using IPv6. Several countries have made IPv6 [mandatory for the public sector](https://www.sidn.nl/en/news-and-blogs/governments-everywhere-make-ipv6-mandatory). In #612 you said that IPv6 support requires a big lift and that it's not on the roadmap. Is there any good reason to keep kicking the can down the road and risk losing public sector customers due to lack of IPv6?

**Comment by DATEx2** [comment]
+1
